nixos/tests/coder.nix

```diff
@@ -1,7 +1,7 @@
 import ./make-test-python.nix ({ pkgs, ... }: {
   name = "coder";
   meta = with pkgs.lib.maintainers; {
-    maintainers = [ shyim ghuntley ];
+    maintainers = [ ghuntley kylecarbs shyim ];
   };
```

Comment:
```suggestion
  meta.maintainers = pkgs.coder.meta.maintainers;
```

Makes sense to also sync tests with coder package.

Reply:
Commit message should be:

> nixos/tests/coder: sync maintainers with coder package

Reply:
Done!

Reply:
Now we wait CI...

Reply:
I said `with` but English is not my native language.
In case, you can word it better, feel free. I wondered if I should have used `to` there.
The key point there was the category being `nixos/tests/coder:`.

Reply:
I'll adjust to `to`, no biggie!